.golangci.yml

```diff
@@ -8,6 +8,7 @@ linters:
   enable:
     - errcheck
     - gofmt
+    - revive
```

Comment:
revive is already in the list, below (I sorted it in alphabetical order)

Reply:
```suggestion
```